src/gt4py/cli.py

```diff
@@ -0,0 +1,70 @@
+"""Command line interface."""
+import typing
+
+import click
```

Comment:
I haven't seen the cli's inner guts yet, but I like seeing click as a framework

Reply:
👍